src/unix/fs.c

```diff
@@ -79,7 +79,7 @@
     defined(__NetBSD__)
 # include <sys/param.h>
 # include <sys/mount.h>
-#elif defined(__sun) || defined(__MVS__) || defined(__NetBSD__) || defined(__HAIKU__)
+#elif defined(__sun) || defined(__MVS__) || defined(__NetBSD__) || defined(__HAIKU__) || defined(__QNX__)
```

Comment:
Can you wrap the lines at 80 characters (it looks like they had already crept past 80 characters prior to this change).